src/pages/guides/configuration/index.md

```diff
@@ -283,6 +283,51 @@ The `.env` file is used to store:
 2. environment variables available to `hooks`.
 3. auto generated secrets used by the `aio` CLI, prefixed by `AIO_`, those should not be edited.
 
+```
+# User secrets
+VARIABLE=value
+
+# Auto-generated secrets
+AIO_runtime_namespace=
+AIO_runtime_auth=
+AIO_runtime_apihost=
+```
+
+### Using environment variables in frontend
+
+Environment variables set in .env can be accessed directly via `process.env`:
+
+```jsx
+<View variable={process.env.VARIABLE}></View>
+```
+
+### Using environment variables in Runtime actions 
+
+Environment variables set in .env need to be passed as inputs to an action and then are available via the action parameters.
+
+#### app.config.yaml
+```yaml
+runtimeManifest:
+   packages:
+     myapp:
+       actions:
+         generic:
+           function: src/myapp/actions/generic/index.js
+           web: 'yes'
+           runtime: nodejs:18
+           inputs:
+               VARIABLE: $VARIABLE
```

Comment:
defining variables called variables makes the documentation too abstract imho,  using something concrete goes a long way to helping users understand this.

DEFAULT_LIMIT=52

Reply:
good call out, just pushed a change making this an example about enable a dev debug mode